src/core/initialize.rs

```diff
@@ -76,12 +76,24 @@ impl PartialOrd for StackFrame {
     }
 }
 
+#[derive(Debug, PartialEq, Eq, Hash, Clone, Serialize, Deserialize)]
+pub struct StackTrace {
+    pub trace: Vec<StackFrame>,
+    pub pid: pid_t,
```

Comment:
should this be an option? how will the various reporty things handle 0?

(at least deserves comment I think)

Reply:
Made it an option!